components/validator/cmd/main.go

```diff
@@ -23,9 +23,9 @@ func main() {
 	app := cli.NewApp()
 	app.Flags = flags.Flags
 	app.Version = fmt.Sprintf("%s-%s", Version, Meta)
-	app.Name = "validator"
-	app.Usage = "L2Output Submitter"
-	app.Description = "Service for generating and submitting L2 Output checkpoints to the L2OutputOracle contract"
+	app.Name = "kroma-validator"
+	app.Usage = "L2 Output Submitter and Challenger"
+	app.Description = "Service for generating and submitting L2 output checkpoints to the L2OutputOracle contract and detecting invalid L2 outputs"
```

Comment:
1. When seeing `components/batcher/cmd/main.go`, it might be better to add a suffix `Serivce` to `app.Usage`. What do you think?
2. For me, challenge not only detects invalid L2 outputs but also corrects it.

```suggestion
-app.Usage = "L2 Output Submitter and Challenger"
-app.Description = "Service for generating and submitting L2 output checkpoints to the L2OutputOracle contract and detecting invalid L2 outputs"
+app.Usage = "L2 Output Submitter and Challenger Service"
+app.Description = "Service for generating and submitting L2 output checkpoints to the L2OutputOracle contract and detecting and correcting invalid L2 outputs to ensure the integrity of the L2 state."
```

Reply:
I changed `components/node/cmd/main.go` also to unify `app.Usage` and `app.Description`. Check it also plz.